cristin-commons/src/main/java/no/unit/nva/exception/FailedHttpRequestException.java

```diff
@@ -6,6 +6,7 @@
 
 public class FailedHttpRequestException extends ApiGatewayException {
 
+    private static final long serialVersionUID = -2345623546891L;
```

Comment:
Is this needed? it dont see it in other upgrade PR

Reply:
pmd complaining